pandas/tests/tslibs/test_timedeltas.py

```diff
@@ -15,6 +15,15 @@
         (np.timedelta64(14, "D"), 14 * 24 * 3600 * 1e9),
         (Timedelta(minutes=-7), -7 * 60 * 1e9),
         (Timedelta(minutes=-7).to_pytimedelta(), -7 * 60 * 1e9),
+        (Timedelta(seconds=1234e-9), 1234),  # GH43764, GH40946
```

Comment:
what do these produce on *master* before any changes

Reply:
They all produce assertion errors, because the nanoseconds part is missing in the respective instantiated timestamps. Please note that all my additions are floats. (Now I note I must delete the `GH40946` from the comments)